bus-mall.js

```diff
@@ -0,0 +1,109 @@
+// refresh page, .random choseing imgags, images stored in array, <15, create counter/
+//whileloop. event listner < on click refresh page
+
+
+var productNames = ['bag', 'banana', 'bathroom', 'boots', 'breakfast', 'bubblegum', 'chair', 'cthulhu', 'dog-duck', 'dragon', 'pen', 'pet-sweep', 'scissors', 'shark', 'tauntaun', 'unicorn', 'water-can', 'wine-glass'];
+
+var productArr = [];
+
+// Constructor function
+function Product(name) {
+ this.name = name;
+ this.votes = 0;
+ this.path = 'image/' + name + '.jpg';
+ productArr.push(this);
+}
+console.log(this.path);
```

Comment:
When does this console log run? Does it run when the Product function is called? Or does it print `undefined` once?